test/protobuf/from_multiple_files_test.exs

```diff
@@ -0,0 +1,28 @@
+defmodule Protobuf.FromMultipleFiles.Test do
+  use Protobuf.Case
+
+  test "generates all messages nested under TopLevel" do
+    defmodule TopLevel do
+      use Protobuf, from: [Path.expand("../proto/basic.proto", __DIR__),
+                           Path.expand("../proto/import.proto", __DIR__),
+                           Path.expand("../proto/imported.proto", __DIR__)]
+    end
+
+    assert %{reason: "hi"} = TopLevel.Authorization.WrongAuthorizationHttpMessage.new(reason: "hi")
```

Comment:
Are these assertions correct? Looks like you've got a single `=` sign. Also the left hand side appears to be a map. I can see that these do actually assert something but if I'm understanding what I'm seeing it's relying on pattern matching as the failure case with may mean that the `assert` is not actually doing anything.

Is it instead meant to be:

```elixir
assert %TopLevel.Authorization.WrongAuthorizationHttpMessage{reason: "hi"} == TopLevel.Authorization.WrongAuthorizationHttpMessage.new(reason: "hi")
```